plugins/shared/PluginManifest.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json.Serialization;
+
+namespace PluginShared;
+
+/// <summary>
+/// This class represents the OpenAI plugin manifest.
+/// </summary>
+public class PluginManifest
+{
+    /// <summary>
+    /// Manifest schema version
+    /// </summary>
+    [JsonPropertyName("schema_version")]
+    public string SchemaVersion { get; set; } = "v1";
+
+    /// <summary>
+    /// The name of the plugin that the model will use
+    /// </summary>
+    [JsonPropertyName("name_for_model")]
+    public string NameForModel { get; set; } = string.Empty;
+
+    /// <summary>
+    /// Human-readable name of the plugin
+    /// </summary>
+    [JsonPropertyName("name_for_human")]
+    public string NameForHuman { get; set; } = string.Empty;
+
+    /// <summary>
+    /// Description of the plugin that the model will use
+    /// </summary>
+    [JsonPropertyName("description_for_model")]
+    public string DescriptionForModel { get; set; } = string.Empty;
+    /// <summary>
```

Comment:
nit: missing line between properties here 
```suggestion

    /// <summary>
```